src/config/media/pipewire.md

```diff
@@ -45,96 +44,124 @@ Install the `wireplumber` package to use this session manager with PipeWire.
-server instance is running.
+for per-user configurations.
 
-Start the PulseAudio server by running `pipewire-pulse` in a terminal emulator.
+### Testing
 
-To check if the replacement is working correctly, use
-[pactl(1)](https://man.voidlinux.org/pactl.1) (provided by the
-`pulseaudio-utils` package):
+[pipewire(1)](https://man.voidlinux.org/pipewire.1) should be started as a your
+user. To test that PipeWire works, run the `pipewire` command in a terminal
+emulator in your session:
 
 ```
-$ pactl info
-
-[...]
-Server Name: PulseAudio (on PipeWire 0.3.18)
-[...]
+$ pipewire
 ```
 
-Once you confirmed that `pipewire-pulse` works as expected, it's recommended to
-autostart it from the same place where you start PipeWire. Alternatively, the
-`pipewire` configuration can be modified to launch `pipewire-pulse` directly:
+Launching `pipewire` should be sufficient to establish a working PipeWire
+session that uses `wireplumber` for session management.
+
+The status of WirePlumber can be checked with:
 
 ```
-# mkdir -p /etc/pipewire/pipewire.conf.d
-# ln -s /usr/share/examples/pipewire/20-pipewire-pulse.conf /etc/pipewire/pipewire.conf.d/
+$ wpctl status
+PipeWire 'pipewire-0' [0.3.74, ...]
+[...]
 ```
 
-for system configurations, or
+If the [PulseAudio interface](#pulseaudio-interface) was configured, use
+[pactl(1)](https://man.voidlinux.org/pactl.1) (provided by the
+`pulseaudio-utils` package) to ensure it is working properly:
 
 ```
-$ true "${XDG_CONFIG_HOME:=${HOME}/.config}"
-$ mkdir -p "${XDG_CONFIG_HOME}/pipewire/pipewire.conf.d"
-# ln -s /usr/share/examples/pipewire/20-pipewire-pulse.conf "${XDG_CONFIG_HOME}/pipewire/pipewire.conf.d/"
+$ pactl info
+[...]
+Server Name: PulseAudio (on PipeWire 0.3.74)
+[...]
 ```
 
-for per-user configurations.
+### Launching Automatically
+
+Once `pipewire` works as expected, it can be configured to launch when starting
+a graphical session. There are several ways this can be achieved:
+
+- **Use the autostarting mechanism of your desktop environment**: many desktop
+   environments have a way to configure applications and programs to start
+   automatically.
+- **Use XDG Desktop Application Autostart**: many desktop environments also
+   support the [Desktop Application Autostart
+   Specification](https://specifications.freedesktop.org/autostart-spec/autostart-spec-latest.html).
+   The `pipewire` package ships a [Desktop
+   Entry](https://specifications.freedesktop.org/desktop-entry-spec/latest/)
+   file for `pipewire` in `/usr/share/applications`. If your environment
+   supports the Desktop Application Autostart, you can start `pipewire` by
+   symlinking the desktop file to the autostart directory
+   (`/etc/xdg/autostart`). If you are using a desktop environment, window
+   manager, or Wayland compositor that does not support this, a tool like
+   [`dex(1)`](https://man.voidlinux.org/dex.1) can be used to add support for
+   Desktop Application Autostart.
```

Comment:
Here is an example for using AwesomeWM: 
```bash
dex --environment Awesome --autostart --search-paths "~/.config/autostart"
```
Though I don't know how important the `--environment` flag really is.
The `--search-paths` are kind of important, otherwise `dex` will just go ahead and start all kind of stuff.

Reply:
I think `--environment` allows filtering by `XDG_CURRENT_DESKTOP`

Reply:
I'm not sure I want to include an example command here. the manpage is short and sweet and has its own examples.

Reply:
It has a lot of examples that don't apply and I found it rather unintuitive that it starts a whole bunch of stuff of packages that pollute the system autostart folders.

Reply:
included an example